### DISCUSSION TITLE
MIT麻省理工學院和BASF巴斯夫的研究員開發了一種數據驅動系統，以加速研發多種機械性質之 3D 列印材料。

### DISCUSSION THREAD

**Discussion by EricAgumonJou** [1101-GS4719-C/Python-Community-GS4719-C#106] [General]
為了縮短研發新材料所需的時間，麻省理工學院的研究人員開發了一種數據驅動的過程，該過程使用機器學習來優化具有多種特性（如韌性和抗壓強度）的新型 3D 列印材料。

由於新的機器學習系統成本更低，產生的浪費也更少，且比人工發現方法更具有創新性，顧運用AI研發新材料是未來的趨勢。

資料文獻: https://scitechdaily.com/mit-uses-ai-to-accelerate-the-discovery-of-new-materials-for-3d-printing/

**Comment by EricAgumonJou** [comment]
優化發現新型材料過程:
在研究人員開發的機器學習系統中，可優化演算法並降低研發失誤率。

材料開發人員選擇一些成分或材料，將其化學成分的詳細資訊輸入演算，並定義新材料應具有的機械性能。 接著演算法會增加和減少這些組件的數量（例如轉動放大器上的旋鈕），並在達到理想組合之前檢查且學習每個公式如何影響材料的屬性。

**Comment by EricAgumonJou** [comment]
另外可額外通過自動化設計，進一步加速該研發過程。 雖然現在幾乎都是研究人員手動混合和測試每個樣品，但外來機器人可以在未來版本的系統中操作分配和混合系統。

**Reply by enoch-lightning** [reply]
現在真是跨領域整合的時代。材料原本應該是機械、化工或材料工程才會觸及的領域—大多是無機化學，現在也能結合熱門的機器學習，加以增進開發速率。:sunglasses:
